src/unix/udp.c

```diff
@@ -263,16 +263,30 @@ static void uv__udp_sendmsg(uv_udp_t* handle) {
  * are different from the BSDs: it _shares_ the port rather than steal it
  * from the current listener.  While useful, it's not something we can emulate
  * on other platforms so we don't enable it.
+ *
+ * zOS does not support getsockname with SO_REUSEPORT option when using 
+ * AF_UNIX  
```

Comment:
nit: punctuation.

Reply:
'.' has been added to the end of the sentence.